evansmusomi/burger-builder
# Initializes Ingredients from the Server

*What this does*
- Fetches the ingredients from the server async
- Initializes fetched ingredients in the burger builder container

*How to verify*
- A user can build a burger as before
- A developer can verify using dev tools that ingredients are fetched from the server and set in the redux store
